drivers/flash/flash_stm32h7x.c

```diff
@@ -33,11 +35,27 @@ LOG_MODULE_REGISTER(LOG_DOMAIN);
 #error Flash driver on M4 core is not supported yet
 #endif
 
-#if defined(CONFIG_MULTITHREADING)
+#define REAL_FLASH_SIZE   KB(LL_GetFlashSize())
+#define SECTOR_PER_BANK  ((REAL_FLASH_SIZE / FLASH_SECTOR_SIZE) / 2)
+#if defined(DUAL_BANK)
+#define STM32H7_SERIES_MAX_FLASH	2048
+#define BANK2_OFFSET	(KB(STM32H7_SERIES_MAX_FLASH) / 2)
+/* When flash is dual bank and flash size is smaller than Max flash size of
+ * the serie, there is a discontinuty between bank1 and bank2.
```

Comment:
Pointing to the source (e.g. ReferenceManual or ApplicationNote) of this statement with indication of page number or url link would be also necessary from my point of view.

Reply:
There is no clear description of this discontinuity in ReferenceManual or ApplicationNote,
it is more like implicite. Thus no source to point to.